Add initial version of 3scale alert SOPs included on 3scale v2.9 (#9)
3scale/3scale-Operations: sops/alerts/backend_worker_jobs_count_running_high.adoc

:toc:
:toc-placement!:

= ThreescaleBackendWorkerJobsCountRunningHigh

toc::[]

== Description

* This alert will trigger if the number of `backend-worker` jobs is greater than a given threshold

== Troubleshooting

> * Check if `backend-worker` pods might be failing and the jobs are not being consumed at all from redis

Author: ```suggestion
* Check if `backend-worker` pods might be failing and the jobs are not being consumed at all from backend's redis
```

* Check if `backend-worker` pods might not be consuming jobs at the required throughput:
- You may need to consider scaling horinzontally `backend-worker` deployment
- You may need to consider increasing `backend-worker` deployment resources requests/limits

> * Check if `backend-listener` might not be able to deal with the incoming traffic, generating too many jobs at redis (maybe `backend-listener` is being attacked?)

Author: ```suggestion
* Check if the current traffic in `backend-listener` is normal based on the historical data. An unexpected change in the traffic pattern can be legit, and will require scaling up, but it also can be due to an abnormal or malicious traffic. Whatever the reason, increased traffic in `backend-listener` generates more jobs for `backend-worker` to process from backend's redis

```

:toc:
:toc-placement!:

= ThreescaleBackendWorkerJobsCountRunningHigh

toc::[]

== Description

* This alert will trigger if the number of `backend-worker` jobs is greater than a given threshold

== Troubleshooting

* Check if `backend-worker` pods might be failing and the jobs are not being consumed at all from redis
* Check if `backend-worker` pods might not be consuming jobs at the required throughput:
- You may need to consider scaling horinzontally `backend-worker` deployment
- You may need to consider increasing `backend-worker` deployment resources requests/limits
* Check if the current traffic in `backend-listener` is normal based on the historical data. An unexpected change in the traffic pattern can be legit, and will require scaling up, but it also can be due to an abnormal or malicious traffic. Whatever the reason, increased traffic in `backend-listener` generates more jobs for `backend-worker` to process from backend's redis

> 

Author: ```suggestion
```

> * Check if redis is having some issues

Author: ```suggestion
* Check if backend's redis is having issues
```

:toc:
:toc-placement!:

= ThreescaleBackendWorkerJobsCountRunningHigh

toc::[]

== Description

* This alert will trigger if the number of `backend-worker` jobs is greater than a given threshold

== Troubleshooting

> * Check if `backend-worker` pods might be failing (so no consuming any job from redis)

Reviewer: ```suggestion
* Check if `backend-worker` pods might be failing and the jobs are not being consumed at all from redis
```

> * Check if `backend-worker` pods might be failing (so no consuming any job from redis)

Author: ```suggestion
* Check if `backend-worker` pods might be failing:
- This can lead to jobs not being consumed at all.
- If left unchecked, backend's redis could fill up
```

> * Check if `backend-worker` pods might not be consuming jobs at the required speed (too much cpu usage causing slowliness?)

Reviewer: ```suggestion
* Check if `backend-worker` pods might not be consuming jobs at the required throughput:
```
